src/index.html

```diff
@@ -183,6 +183,79 @@ <h2 class="h2">Next event</h2>
         </filter>
       </svg>
     </section>
+
+    <section class="bg-light-gray tablet:mx-4 my-16">
+      <div class="max-w-6xl mx-auto text-center px-4 py-20">
+        <h2 class="h2">Featuring talent from...</h2>
```

Comment:
Could we swap `...` with `…` - yep they look 100% identical in the Tiempos font, but it's slightly nicer for people with screen readers to use the ["proper" ellipsis character](https://www.compart.com/en/unicode/U+2026)